<!-- 
  Example launch file for octomap_server mapping: 
  Listens to incoming PointCloud2 data and incrementally builds an octomap. 
  The data is sent out in different representations. 

  Copy this file into your workspace and adjust as needed, see
  www.ros.org/wiki/octomap_server for details  
-->
<launch>
	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
		<param name="resolution" value="0.06" />
		
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="map" />
		
		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model/max_range" value="5.0" />

		<param name="pointcloud_min_z" value="0.0" />
		<param name="pointcloud_max_z" value="5.0" />
		<!-- filter ground -->
		<param name="filter_ground" value="true" />

		<!-- filter ground distance -->
		<param name="ground_filter/distance" value="0.3" />
		<param name="ground_filter/plane_distance" value="0.3" />


		<!-- data source to integrate (PointCloud2) -->
		<remap from="cloud_in" to="/rgbd_pcl" />
	
	</node>
</launch>
